trtools/testsupport/sample_vcfs/beagle/hipstr_imputed_merge1.vcf

```diff
@@ -0,0 +1,3416 @@
+##fileformat=VCFv4.2
```

Comment:
Do we still need this file? Doesn't the test instead refer to this file in the mergeSTR_vcfs directory, instead?